src/components/statusBars/StatusBar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { StatusBar, View } from 'react-native'
+
+const MyStatusBar = (props) => {
```

Comment:
:thinking: I'd suggest importing React's `StatusBar` with another name and use the correct name for our component. This name is shown on the logs when debugging.
```javascript
import { StatusBar as ReactStatusBar, View } from 'react-native'

const StatusBar = (props) => {
  // ...
}
// ...
```